gateway/1.6/modules/ROOT/pages/flex-gateway-publish-soap-apis.adoc

```diff
@@ -0,0 +1,18 @@
+= Publishing SOAP APIs
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Flex Gateway does not natively support SOAP APIs and does not provide any schema validation for XML. However, you can publish an HTTP API instance to secure a SOAP upstream service.
+
+To create an HTTP API instance to secure SOAP API upstream service, refer to the relevant tutorial for your running mode:
+
+* xref:flex-gateway-publish-apis-conn.adoc[]
+* xref:flex-gateway-publish-apis-local.adoc[]
+
+== SOAP API Policy Limitations
+
+Flex Gateway policies return JSON error messages when the policy blocks a request. Most SOAP clients cannot read the JSON messages.
+
+The xref:policies-included-schema-validation.adoc[] and xref:policies-included-json-threat-protection.adoc[] do not support SOAP API endpoints.
```

Comment:
I'm thinking these two policies should be in a bulleted list, so it's easier to scan, and so we don't have such a wordy sentence.

Something like:
"The following policies do not support SOAP API endpoints:"
...